packages/stencil/components/pagination/src/global.dev.ts

```diff
@@ -0,0 +1,107 @@
+// ###
+// global script file to include only in dev mode with the www dev server.
+// it always has to include './components' and './global.ts'
+// ###
+
+import './components';
+import './global';
+import { OdsLogger } from '@ovhcloud/ods-core';
+import { OdsInitializedEvent, OdsInitializedEventName, OdsPaginationCurrentChangeEvent } from '@ovhcloud/ods-core';
+import '@ovhcloud/ods-stencil/components/icon';
+import '@ovhcloud/ods-stencil/components/button';
+
+const logger = new OdsLogger('global-dev');
+logger.log('init');
+
+/* wait for the select to be initialized as web component */
+(async () => {
+  await customElements.whenDefined('osds-pagination');
+  await customElements.whenDefined('osds-pagination-group');
+  await customElements.whenDefined('osds-pagination-option');
+})();
+
+document.addEventListener(OdsInitializedEventName, event => {
+  const evt = event as OdsInitializedEvent;
+  const instance = evt.detail.instance;
+  instance.logging(true);
+  logger.log('odsInitialized received');
+});
+
+type ExampleData = {
+  component: HTMLOsdsPaginationElement | null;
+  info: HTMLElement | null;
+};
+
+/**
+ * create an object reference for a given integration example
+ * @param id
+ */
+function getExampleData(id: string): ExampleData {
```

Comment:
This comes from the Select component, to be removed (error in the browser console since it cannot find select1 etc.)